bv-evaluation/collect-data-hdel.py

```diff
@@ -3,16 +3,20 @@
 import numpy as np
 import pandas as pd
 import os 
+from enum import Enum
+from collections import Counter
 
-paper_directory = 'for-paper/'
+output = Enum('output', [('counterexample', 1), ('proved', 2), ('failed', 0)])
+
+paper_directory = ''
 benchmark_dir = "../SSA/Projects/InstCombine/HackersDelight/"
 res_dir = "results/HackersDelight/"
 raw_data_dir = paper_directory + 'raw-data/HackersDelight/'
-reps = 1
+REPS = 1
```

Comment:
Nit: why did the capitalization here have to change?

Reply:
the change was functionally unnecessary, but makes the script consistent with the `compare` script format-wise

Reply:
the change was functionally unnecessary, but makes the script consistent with the `compare` script format-wise